Installing-and-Configuring/OS/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Operating Systems
+permalink: /OS/
+nav_order: 80
+parent: Installing and configuring
+has_children: yes
+has_toc: yes
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Heads and Operating Systems
+====
+
+Since a heads only system would be very limited you will need to install at least one Operating System.  The installation procedures vary greatly depending on which distro you choose, which version of that distro, and if you are encrypting partitions.  Some choices you will have to make include:
+
+* partition table
+* LUKS encryption (yes/no)
+* LUKS version (1 or 2)
+* single vs multiboot
+* simple distro vs hypervisor
+
+All together there are a lot of technical decisions.  This document will attempt to inform you enough to make the decisions.
+
+
+partition table
+----
+
+heads currently requires a separate boot partition which is a common configuration for Linux.  The heads model leaves the /boot partition unencrypted.  This is safe because heads will verify files at boot to detect tampering.
+
+When you partition storage make sure to create /boot or have the OS installer do it for you.  Generally, the boot partition is much smaller than the other OS partition(s).  You may keep everything else on 1 partition to make it easier to manage but the kernel and initrd on the boot partition can mount any number of others--encrypted or not.
```

Comment:
You may keep everything else on one additional partition to make it easier to manage, but the kernel and initrd is required to be under the seperated boot partition. Linux initrd will normally have the ability to mount the other partitions--encrypted or not.

Reply:
reworded.  I had a typo in the original text which made it confusing.  I combined your text with mine. push in a few...